### PR TITLE
Add map clustering

### PR DESCRIPTION
**Pull request by chodorowicz**
solves: https://github.com/lightning-team/Ekostraz/issues/74

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -34,6 +35,7 @@
     "@angular/platform-browser-dynamic": "^8.2.14",
     "@angular/router": "^8.2.14",
     "@sentry/browser": "^5.6.3",
+    "js-marker-clusterer": "^1.0.0",
```
Comment (CorneeSean) [inline frontend/package.json]: Just curious, do we need this package as well for @agm/js-marker-clusterer or is it a leftover from your prototyping? Doesn't the @agm/js-marker-clusterer manage it's own dependencies?

File: frontend/src/app/modules/interventions/map/map.component.html [hunk 2]
```diff
@@ -6,14 +6,16 @@
...
-      [latitude]="intervention.geoLat"
-      [longitude]="intervention.geoLng"
-    >
-      <agm-info-window [disableAutoPan]="false">
-        <app-intervention-details [intervention]="intervention" embedded="true"> </app-intervention-details>
-      </agm-info-window>
-    </agm-marker>
+    <agm-marker-cluster imagePath="/assets/m">
+      <agm-marker
+        *ngFor="let intervention of interventions.slice(0, 200)"
```
Comment (CorneeSean) [inline frontend/src/app/modules/interventions/map/map.component.html]: Not the goal of this PR, but I am not sure if we still need `interventions.slice(0, 200)`. This was added when we didn't have clustering and we had 1000 records of FAKE_DATA provided to the map at once, so the map got blocked for a long time before loading ;) Could we maybe clean this up?

File: frontend/package.json (hunk 1 above)
Comment (chodorowicz) [reply frontend/package.json]: It's needed as peer dependency. It's mentioned in the README.md and it fails without this dependency. I was also a bit surprised.
https://github.com/SebastianM/angular-google-maps/tree/master/packages/js-marker-clusterer#installation

File: frontend/src/app/modules/interventions/map/map.component.html (hunk 2 above)
Comment (chodorowicz) [reply frontend/src/app/modules/interventions/map/map.component.html]: Will do!

Comment (chodorowicz) [reply frontend/src/app/modules/interventions/map/map.component.html]: OK, done

**Comment by CorneeSean** [comment]
👌👌

**Pull request merged**